### DISCUSSION TITLE
Access Control RBAC

### DISCUSSION THREAD

**Discussion by tiklein404** [totaljs/flow#117] [General]
Hi Guys, 
is there any kind of RBAC for the flows?
Or can anyone see anything?
Thank you and best, IS

**Comment by petersirka** [comment]
Sorry for the delay. The current version does not implement RBAC, but implementing it isn't a problem. You need to think about how you want to implement it deeply. It means if you want to apply RBAC directly to the WebSocket communication or if it's enough to implement it only for e.g. listing FlowStreams.
